### PR TITLE
Add pretty printing functionality for lerchphi fuction

### LINKED ISSUE #6013: Pretty printing for lerchphi, polylog
The gsoc-3 branch adds various new special functions, some of which might profit from pretty printing. In particular: lerchphi, polylog.

I did not add pretty printing for lerchphi, since printing it as a capital phi seemed like a potential source of confusion to me (e.g. the distribution function of the standard normal random variable is also often written as capital phi).

I did not add printing for polylog since this typically written with one of the variables as subscript (i.e. there is some non-trivial work to do for printing).

None of this should be difficult, once it has been decided what to print how.

 - [x] Add printing for lerchphi
 - [ ] Add printing for polylog

Original issue for #6013: http://code.google.com/p/sympy/issues/detail?id=2914
Original author: https://code.google.com/u/104531927090589914088/

### PR DESCRIPTION
**Pull request by bhavya17037**
#### Brief description of what is fixed or changed
Added functionality to pretty print lerchphi function in pretty.py
Fixes the lerchphi part of #6013.

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by bhavya17037** [comment]
@jksuom 
Hi, I fixed what you had told in my previous pull request (which i had earlier closed).
Is this change okay?

Thanks!

File: sympy/printing/pretty/pretty_symbology.py [hunk 1]
```diff
@@ -127,6 +127,8 @@ def xstr(*args):
 # aliases
 greek_unicode['lambda'] = greek_unicode['lamda']
 greek_unicode['Lambda'] = greek_unicode['Lamda']
+greek_unicode['lerchphi'] = greek_unicode['phi']
```
[the remark this answers is not in the document]
Comment (bhavya17037) [reply sympy/printing/pretty/pretty_symbology.py]: I removed those lines, but then lerchphi function was not pretty printed correctly. When I added them back, they were printing correctly.

**Comment by bhavya17037** [comment]
@jksuom 
Hi!
I changed the line where lerchphi was being assigned the symbol of lowercase phi,
and changed it to uppercase Phi.

It is in my latest commit. :)
Could you review it and tell me if it is fine? :)

File: sympy/printing/pretty/pretty_symbology.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (bhavya17037) [reply sympy/printing/pretty/pretty_symbology.py]: Hi,
I removed the lambda lines, but then i was'nt able to pretty print lambda.
Could you see my most recent commit, in that i am able to pretty print lambda using "Lambda" and "lambda", also, i am able to pretty print lerchphi function using "Lerchphi" and "lerchphi"

[the remark this answers is not in the document]
Comment (jksuom) [reply sympy/printing/pretty/pretty_symbology.py]: I did not mean the lambda lines. Only this line and the next one.

[the remark this answers is not in the document]
Comment (bhavya17037) [reply sympy/printing/pretty/pretty_symbology.py]: But without those, I am not able to pretty print the lerchphi function.
Also, how would do I get this commit merged? I am very new to open source development, so I am not very familiar as to how things go around here :))

File: sympy/printing/pretty/pretty.py [hunk 2]
```diff
@@ -1266,11 +1266,13 @@ def _print_Function(self, e, sort=False, func_name=None):
     def _special_function_classes(self):
         from sympy.functions.special.tensor_functions import KroneckerDelta
         from sympy.functions.special.gamma_functions import gamma, lowergamma
+        from sympy.functions.special.zeta_functions import lerchphi
         from sympy.functions.special.beta_functions import beta
         from sympy.functions.special.delta_functions import DiracDelta
         from sympy.functions.special.error_functions import Chi
         return {KroneckerDelta: [greek_unicode['delta'], 'delta'],
                 gamma: [greek_unicode['Gamma'], 'Gamma'],
+                lerchphi: [greek_unicode['lerchphi'], 'lerchphi'],
```
Comment (jksuom) [inline sympy/printing/pretty/pretty.py]: This could be `greek_unicode['Phi']`.

**Comment by bhavya17037** [comment]
@jksuom 
Hi, 
changed the greek_unicode['lerchphi'] to greek_unicode['Phi'] in my latest commit :)
What should I do next?

**Comment by asmeurer** [comment]
This change should have a release note entry.

**Comment by asmeurer** [comment]
In the future, start from master when creating a new branch for a change. This branch contains the change from https://github.com/sympy/sympy/pull/15022.

**Comment by oscargus** [comment]
Continued in #15971

**Pull request merged**